src/polar/PolarAngleAxis.tsx

```diff
@@ -44,12 +47,27 @@ export interface PolarAngleAxisProps extends PropsInjectedFromRedux {
   reversed: boolean;
   dataKey?: DataKey<any>;
   tick?: SVGProps<SVGTextElement> | ReactElement<SVGElement> | ((props: any) => ReactElement<SVGElement>) | boolean;
+  scale: ScaleType | RechartsScale;
+  type?: 'category' | 'number'; // so there is code that checks if angleAxis.type is number but it actually never behaves as a number
```

Comment:
Since we're hardcoding this can we just remove it?

Reply:
If I remove it then various other places in the ChartUtils break. I'll leave it here for now.